src/App.js

```diff
@@ -1,26 +1,17 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
-import { Route, Routes } from 'react-router-dom';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
 import './App.css';
 import Navbar from './components/Navbar';
-import MyProfile from './components/My-Profile';
+import MissionList from './components/Missions/MissionList';
 
-class App extends React.Component {
-  render() {
-    return (
-      <div>
-        <Navbar />
-        <main>
-          <Routes>
-            {/* <Route path="/" element={<Rockets />} />
-            <Route path="/dragons" element={<Dragons />} />
-            <Route path="/missions" element={<Mission />} /> */}
-            <Route path="/my-profile" element={<MyProfile />} />
-          </Routes>
-        </main>
-      </div>
-    );
-  }
-}
+const App = () => (
+  <BrowserRouter>
+    <Navbar />
+    <Routes>
+      <Route path="Missions" element={<MissionList />} />
+    </Routes>
```

Comment:
- [ ] Kindly be careful when you merge with the development branch. You have deleted the route for the "My profile" page, make sure you keep that line from the development branch